apps/vs-agent/src/utils/WebVhDidRegistrar.ts

```diff
@@ -55,15 +121,26 @@ export class WebVhDidRegistrar implements DidRegistrar {
    */
   public async update(agentContext: AgentContext, options: WebVhDidUpdateOptions): Promise<DidUpdateResult> {
     try {
-      const { did, domain, didDocument: inputDidDocument } = options
+      const { did, didDocument: inputDidDocument } = options
```

Comment:
Why not just call it `didDocument`? The `inputDidDocument` was overridden because it was used to identify the `didDocument` input object. The `didDocument` variable should be more consistent with the result of the create method
```suggestion
      const { did, didDocument } = options
```